CorvallisBus.Core/Models/RouteArrivalsSummary.cs

```diff
@@ -124,5 +124,27 @@ public static string ToScheduleSummary(List<BusArrivalTime> arrivals, DateTimeOf
                 return "Last arrival at " + lastTimeDescription;
             }
         }
+
+        public override bool Equals(object obj) => obj is RouteArrivalsSummary other && Equals(other);
```

Comment:
all this is just for better test writing and better error messages when tests fail.